src/resolve/resolved-dns-zone.c

```diff
@@ -78,12 +78,13 @@ void dns_zone_flush(DnsZone *z) {
 }
 
 DnsZoneItem* dns_zone_get(DnsZone *z, DnsResourceRecord *rr) {
-        DnsZoneItem *i;
+        DnsZoneItem *first;
 
         assert(z);
         assert(rr);
 
-        LIST_FOREACH(by_key, i, hashmap_get(z->by_key, rr->key))
```

Comment:
the fact that function calls cannot be inlined anymore also needs to be documented in the list header I think

Reply:
If it's still possible, maybe do that to keep the diff to a minimum? There's other instances as well when a temporary new variable was added to avoid the inline function call